### PR TITLE
chore(eslint): fix eslint for intellij by using absolute path for tsconfig.json

### PR DESCRIPTION
**Pull request by andrestone**
### Commit Message
chore(eslint): fix eslint for intellij by using absolute path for tsconfig.json

This is a less invasive fix for enabling ESLint support for IntelliJ IDE's. It works by simply changing the reference for `tsconfig.json` to use an absolute path before re-exporting the ESLint config in each package's `.eslintrc.js` file.

Related:
#7423

### End Commit Message

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by andrestone** [comment]
ping @nija-at

**Comment by andrestone** [comment]
Done. Could you test it on VSCode? If this is merges, I guess this has to be updated (removed):

>If you're using the VS Code and would like to see eslint violations on it, install the eslint extension. The VS Code setting needed for the extension to work on the monorepo is configured in the folder settings.

**Comment by nija-at** [comment]
> Done. Could you test it on VSCode? If the PR gets merged, I guess this has to be updated (removed):
> 
> > If you're using the VS Code and would like to see eslint violations on it, install the eslint extension. The VS Code setting needed for the extension to work on the monorepo is configured in the folder settings.

This fixes it up for VS Code as well. Very cool! 
I believe you can also delete this folder - `.vscode/`.

**Comment by andrestone** [comment]
> [#7880 (comment)](https://github.com/aws/aws-cdk/pull/7880#issuecomment-626786350)

Done! Thanks for reviewing.

File: packages/@aws-cdk/alexa-ask/.eslintrc.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 const baseConfig = require('../../../tools/cdk-build-tools/config/eslintrc');
+baseConfig.parserOptions.project = __dirname + "/tsconfig.json";
```
Comment (jogold) [inline packages/@aws-cdk/alexa-ask/.eslintrc.js:2]:
```suggestion
baseConfig.parserOptions.project = __dirname + '/tsconfig.json';
```
use the quotes enforced by the eslint config? 😄

Comment (andrestone) [reply packages/@aws-cdk/alexa-ask/.eslintrc.js:2]: Heh, shoemaker's son always goes barefoot :)

**Pull request merged**